fuzz/fsfuzzer.py.data/attrfix.patch

```diff
@@ -0,0 +1,20 @@
+diff -crB a/fstest.c b/fstest.c
```

Comment:
Where these patches comes from? Would you please send them to https://github.com/stevegrubb/fsfuzzer so eventually we can remove them?